packages/@poupe-css/src/rules.ts

```
@@ -299,6 +247,159 @@ function atRuleException(key: string, value: CSSRulesValue): boolean {
+    } else if (typeof value === 'number') {
+      // Apply kebab-case conversion only to properties, like formatCSSProperties
+      yield `${prefix}${mayNormalize(key)}: ${value};`;
+    } else if (typeof value === 'string') {
+      // string, omit empty
+      if (value) {
+        // Apply kebab-case conversion only to properties, like formatCSSProperties
+        yield `${prefix}${mayNormalize(key)}: ${value};`;
+      }
+    } else if (Array.isArray(value)) {
+      if (value.length === 0) {
+        continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)